packages/coinstac-ui/app/render/components/common/runs-list.jsx

```diff
@@ -8,6 +8,20 @@ function getActiveTime(hoursSinceActive) {
   return Date.now() - (hoursSinceActive * 60 * 60 * 1000);
 }
 
+const isUserA = (userId, groupArr) => {
```

Comment:
probably worth putting this somewhere resuable

Reply:
Good idea. I've put it in coinstac-ui/app/render/utils/helpers.js

Reply:
👍